submissions/odunayoFeyiAkinbile/HTML/product-page.html

```diff
@@ -9,6 +9,8 @@
         <h1>FAME</h1>
     </header>
     <nav>
+        <a href="../html/contact-form.html">Contact Form</a>
+        <a href="../html/product-page.html">Product</a>
```

Comment:
This link wont work. The file is in the HTML folder not 'html'

Reply:
i have made the changes